socket.js

```diff
@@ -0,0 +1,14 @@
+var server = require('http').Server();
+var io = require('socket.io')(server);
+
+var Redis = require('ioredis');
+var redis = new Redis();
+
+redis.psubscribe('*');
+
+redis.on('pmessage', function(subscribed, channel, message) {
+    message = JSON.parse(message);
+    io.emit(channel, message.data);
```

Comment:
you should emit the entire message... The Event Itself is ok to go to client too... As we can have multiple events on same channel, for future.